tests/spark-it/src/test/resources/log4j2-test.xml

```diff
@@ -21,7 +21,7 @@
         <Console name="stdout" target="SYSTEM_OUT">
             <PatternLayout pattern="%d{yy/MM/dd HH:mm:ss,SSS} %p [%t] %c{1}: %m%n%ex"/>
             <Filters>
-                <ThresholdFilter level="FATAL"/>
+                <ThresholdFilter level="ERROR"/>
```

Comment:
Does `flink-it` also need to be consistent?

https://github.com/apache/incubator-celeborn/blob/c4135dc1b1e8fabf535da63107e66072c408885c/tests/flink-it/src/test/resources/log4j2-test.xml#L19-L32

Reply:
updated. cc @FMX

Reply:
nit

https://github.com/apache/incubator-celeborn/blob/c4135dc1b1e8fabf535da63107e66072c408885c/tests/flink-it/src/test/resources/log4j2-test.xml#L19C16-L19C30

https://github.com/apache/incubator-celeborn/blob/c4135dc1b1e8fabf535da63107e66072c408885c/tests/flink-it/src/test/resources/log4j2-test.xml#L32C15-L32C28